src/test/java/edu/kis/powp/jobs2d/VisitorTest.java

```diff
@@ -1,47 +1,33 @@
 package edu.kis.powp.jobs2d;
 
-import edu.kis.powp.jobs2d.Job2dDriver;
-import edu.kis.powp.jobs2d.command.DriverCommand;
-import edu.kis.powp.jobs2d.command.OperateToCommand;
-import edu.kis.powp.jobs2d.command.SetPositionCommand;
 import edu.kis.powp.jobs2d.visitor.DriverCommandVisitor;
 import edu.kis.powp.jobs2d.visitor.IVisitor;
 import edu.kis.powp.jobs2d.visitor.OperateToVisitable;
 import edu.kis.powp.jobs2d.visitor.SetPositionVisitable;
-import edu.kis.powp.jobs2d.visitor.Visitor;
+import edu.kis.powp.jobs2d.visitor.VisitorDriverOperation;
 
 /**
  * Visitor interface test.
  * 
  * @author Adrian
  */
 
-class VisitorT extends Visitor{
-    @Override
-    public void visit(DriverCommand command, Job2dDriver driver) {
-        if(command instanceof OperateToCommand)
-            System.out.println("Operate to execution");
-        if(command instanceof SetPositionCommand)
-            System.out.println("Set position execution");
-        
-        super.visit(command, driver);
-    }
-}
-
-class EmptyDriver implements Job2dDriver{
+class PrintOperationDriver implements Job2dDriver{
     @Override
     public void setPosition(int x, int y) {
+        System.out.println("setPosition("+x+", "+y+")");
     }
     @Override
     public void operateTo(int x, int y) {
+        System.out.println("operateTo("+x+", "+y+")");
     }
 }
 
 public class VisitorTest {
```

Comment:
CommandVisitorTest (there may be few different visitor hierarchies in the project!)